tailwind.config.js

```diff
@@ -18,6 +18,8 @@ module.exports = {
     extend: {
       colors: {
         themeGray: '#121212',
+        gray: 'rgba(255, 255, 255, 0.5)',
```

Comment:
I don't think you need a theme value for this one. I'd just use `text-white/50` or `bg-white/50`